spiketoolkit/postprocessing/postprocessing_tools.py

```diff
@@ -480,20 +482,14 @@ def compute_pca_scores(recording, sorting, unit_ids=None, n_comp=3, by_electrode
     if verbose:
         print("Fitting PCA of %d dimensions on %d waveforms" % (n_comp, len(all_waveforms)))
 
-    pca = PCA(n_components=n_comp, whiten=True)
-    pca.fit_transform(all_waveforms)
-    scores = pca.transform(all_waveforms)
+    pca = PCA(n_components=n_comp, whiten=False)
+    pca.fit(all_waveforms)
```

Comment:
maybe the **whitten** could be expose at the method level.
In case of already whitten signals.

Reply:
sounds good

Reply:
BTW, I found out that the pca.scores returned by PCA are not the same as the projections of the waveforms on the different eigenvectors...I spent almost the entire day yesterday trying to understand why..In the end I just substituted the pca_scores calculation with the manual projection